data/pilots/first-order/tie-wi-whisper-modified-interceptor.json

```diff
@@ -101,10 +146,28 @@
         "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
       },
       "text": "Commanded by Supreme Leader Kylo Ren, the 709th Legion fights without mercy or remorse to crush the Resistance and bring the galaxy under the First Order's dominion.",
-      "hyperspace": false,
-      "cost": 0,
-      "slots": ["Talent", "Talent", "Missile", "Tech", "Configuration"],
-      "ffg": -999,
+      "hyperspace": true,
+      "cost": 47,
+      "slots": ["Talent", "Talent", "Missile", "Tech", "Tech", "Configuration"],
+      "ffg": 923,
+      "artwork": "???",
+      "image": "???",
+      "keywords": ["TIE"]
+    },
+    {
+      "name": "Red Fury Zealot",
+      "xws": "redfuryzealot",
+      "initiative": 2,
+      "limited": 0,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "text": "Also called Red Fury, the 709th Legion is charged by Supreme Leader Kylo Ren with rooting out dissenters at Black Spire Outpost.",
+      "hyperspace": true,
+      "cost": 44,
+      "slots": ["Talent", "Talent", "Missile", "Tech", "Tech", "Configuration"],
```

Comment:
Extra Talent slot added to Red Fury Zealot. Should be ["Talent", "Missile", "Tech", "Tech", "Configuration"]